.orchestra/config/components/revng-theme.yml

```diff
@@ -0,0 +1,35 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_theme_args():
+repository: revng-vscode-theme
+license: LICENSE.md
+binary_archives: private
+build_dependencies:
+- revng-js-prerequisites
+configure: |
+  mkdir -p "$BUILD_DIR"
+  cd "$BUILD_DIR"
+  cp "$SOURCE_DIR/"package{,-lock}.json .
+  "$ORCHESTRA_DOTDIR"/support/js-populate-cache npm -o ./.cache
+install: |
+  cd "$BUILD_DIR"
+  rsync -aq \
+    --exclude=.git --exclude-from=<(git -C "$SOURCE_DIR" ls-files --exclude-standard -oi --directory) \
+    "$SOURCE_DIR/" .
+  export NODE_CACHE="$BUILD_DIR/.cache:$NODE_CACHE"
+  npm install --omit peer
+  # vsce will try to do an update check if in a tty, we use cat to suppress this
+  node_modules/.bin/vsce package --no-dependencies | cat
```

Comment:
* Can we use `npm run package`?
* Why `--no-dependencies`?

Reply:
Added `--no-dependencies` to the `package.json` of the respective packages. Updated orchestra to use `npm run package`.